src/utils/messages.js

```diff
@@ -54,13 +54,11 @@ export default {
 
   ERR_LOADING_AUDIO({ attr, url, error }) {
     return (
-      <p>
-        Error while loading audio. Check <code>{attr}</code> field in task.
-        <br />
-        Technical description: {error}
-        <br />
-        URL: {url}
-      </p>
+      <div>
```

Comment:
```suggestion
      <div data-testid="error:audio">
```

Maybe we make this explicit so we can update the checks for errors in test to be explicit. That way we know for sure this error is getting shown, not just the one in the Audio interface itself.